cpp/src/transform/row_bit_count.cu

```diff
@@ -334,10 +334,21 @@ template <>
 __device__ size_type row_size_functor::operator()<string_view>(column_device_view const& col,
                                                                row_span const& span)
 {
-  column_device_view const& offsets = col.child(strings_column_view::offsets_column_index);
   auto const num_rows{span.row_end - span.row_start};
+  if (num_rows == 0) {
+    // For empty columns, the `span` cannot have a row size.
+    return 0;
+  }
+
+  auto const& offsets = col.child(strings_column_view::offsets_column_index);
   auto const row_start{span.row_start + col.offset()};
   auto const row_end{span.row_end + col.offset()};
```

Comment:
Humm, are you sure this is correct? The starting row may be plus offset, but the end row should typically not. Unless this is something else.

Reply:
Yes.  `row_start` and `row_end` are offset agnostic row numbers.  To get the correct indices in the offsets column, you have to shift by `.offset()`.  As usual, the dual meanings of ".offset()" vs "offsets" confuses things :)

Reply:
We should rename `.offset()` to something else.  Maybe `.parallax()`. I kid :)